2/contrib/s2i/run

```diff
@@ -186,12 +186,21 @@ if [ ! -e ${JENKINS_HOME}/configured ]; then
     touch ${JENKINS_HOME}/configured
 else  
   if [ ! -z "${OVERRIDE_PV_CONFIG_WITH_IMAGE_CONFIG}" ]; then
-    echo "Overriding jenkins config.xml stored in ${JENKINS_HOME}/config.xml"
-    rm -f ${JENKINS_HOME}/config.xml
-	
-    create_jenkins_config_xml
+    echo "Overriding jenkins configuration stored in ${JENKINS_HOME}"
+
+    # When overriding config, we should override every config file except plugins which might be a special case (?)
```

Comment:
# remove any files/directories in $JENKINS_HOME that are specified in $image_config_dir, otherwise the two will get merged when cp is run below